rebar.lock

```diff
@@ -0,0 +1,8 @@
+[{<<"jiffy">>,
+  {git,"git@github.com:kivra/jiffy.git",
+       {ref,"9ea1b35b6e60ba21dfd4adbd18e7916a831fd7d4"}},
+  0},
+ {<<"jsx">>,
```

Comment:
Is this used? I didn't see it anywhere.

Reply:
no, I was changing this back and forth, will remove it